DESIGN.md

```diff
@@ -325,18 +325,99 @@ In the addition of the previous rules, an operator for `Flowable`:
 
 ### Creation
 
-Creation of the various types should be exposed through factory methods that provide safe construction.
+Unlike RxJava 1.x, 2.x base classes are abstract and stateless and generally no longer wrap an `OnSubscribe` callback - this saves allocation in assembly time without limiting the expressiveness. Operator methods and standard factories still live as final on the base classes. 
```

Comment:
> base classes are abstract and stateless and generally no longer wrap an `OnSubscribe` callback

This paragraph suggests a design that is already implemented. This design document is intended to capture the requirements for 2.x, what **needs** to be implemented for 2.x without the presumption that an implementation has been chosen or finalized. I'm fine with mentioning this aspect of the design but could you please elaborate this paragraph so that everyone can be on the same page?